src/views/virtualmachines/details/tabs/diagnostic/virtual-machine-diagnostic-tab.scss

```diff
@@ -2,3 +2,17 @@
   display: flex;
   justify-content: space-between;
 }
+
+.VirtulMachineDiagnosticTabRow--expended {
```

Comment:
You have a couple of typos in the class name: 'Virtul' -> 'Virtual' and 'expended' -> 'expanded'

Reply:
thanks!